.eslintrc.json

```diff
@@ -22,10 +25,32 @@
   ],
 
   "rules": {
-    "comma-dangle": ["error", "only-multiline"],
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "linebreak-style": ["error", "unix"],
-    "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "indent": [
+      2, 
+      4
+    ],
+    "comma-dangle": [
+        "error", 
+        "never"
+    ],
+    "no-unused-vars": [
+        "error", 
+        {
+            "vars": "local", 
+            "args": "none" 
+        }
+    ],
+    "arrow-parens": [
+        2, 
+        "as-needed"
+    ],
+    "no-shadow": "off",
+    "consistent-return": "off",
+    "prefer-promise-reject-errors": "off",
+    "arrow-body-style": "off",
```

Comment:
Wew shouldn't turn this rule off. this will help our es6 function definitions be as concise as possible. It might be better to change it to as-needed.